src/css.rs

```diff
@@ -89,7 +89,14 @@ button.light-green {
 button.blue {
     background-color: #0000FF;
     color: #FFFFFF;
-    font-size: 80px;
+    font-size: 30px;
```

Comment:
Why is this bigger than the rest of the fonts?

Reply:
I was just playing around with them, no font-size is finalized yet

Reply:
ok, this can also be fixed in a follow up PR